src/target/adiv5.c

```diff
@@ -983,6 +983,16 @@ void adiv5_dp_init(adiv5_debug_port_s *const dp)
 				if (target_ap == ap)
 					target_halt_resume(target, false);
 			}
+
+			/*
+			* Due to the Tiva TM4C1294KCDT repeating the single AP ad-nauseum, this check is needed
+			* so that we bail rather than repeating the same AP ~256 times.
+			*/
```

Comment:
These should have a ' ' (space) inserted just before the front of line * please

Reply:
Done; I accidentally messed that up when I was retabbing the file as my editor had used spaces :facepalm:

Reply:
Easily done! We have to fix it every time we change line indentation in VSCode with block selections..